.generator/src/generator/templates/api.j2

```diff
@@ -241,51 +239,26 @@ localVarQueryParams.Add("{{ parameter.name }}", parameterToString(*r.{{ name|var
 	{%- endif %}
 	{%- endfor %}
 	
-	{%- for name, parameter in operation|parameters if parameter.in == "form" %}
-	{#isFile#}
-
-	localVarFormFileName = "{{ name }}"
-	{%- if parameter.required %}
-	localVarFile := *r.{{ name|variable_name}}
+	{%- if formParameter %}
+        formFile := FormFile{}
```

Comment:
I don't know? We use this in a single place right now, and mostly because we don't support XML file posting. It actually would be easier with my PR as we can easily add arrays to the struct.